src/components/Form.js

```diff
@@ -0,0 +1,10 @@
+const Form = () => (
+  <>
+    <form>
+      <input type="text" placeholder="Book Title" />
+      <input type="text" placeholder="Author Name" />
+    </form>
```

Comment:
- [ ] Please make sure that your form component has a button to submit the button wouldn't add the book *yet*, this is just a placeholder for now :pray: